rero_ils/modules/acq_accounts/jsonschemas/form_acq_accounts/acq_account-v0.0.1.json

```diff
@@ -0,0 +1,75 @@
+[
+  {
+    "type": "fieldset",
+    "items": [
+      {
+        "title": "Library",
+        "key": "library.$ref",
+        "type": "select",
+        "remoteRecordType": "libraries",
+        "remoteRecordFiltersByOwningOrganisation": "pid",
+        "readonly": "true"
+      }
+    ]
+  },
+  {
+    "type": "fieldset",
+    "items": [
+      {
+        "key": "name",
+        "required": true,
+        "validationMessage": {
+          "alreadyTakenMessage": "The acquisition account name is already taken"
+        },
+        "remoteRecordType": "acq_accounts"
+      }
+    ]
+  },
+  {
+    "type": "fieldset",
+    "items": [
+      {
+        "key": "Description"
+      }
+    ]
+  },
+  {
+    "type": "fieldset",
+    "items": [
+      {
+        "key": "currency",
+        "type": "select",
+        "titleMap": {
+          "CHF": "Swiss Franc",
+          "EUR": "Euro",
+          "USD": "American Dollar"
+        },
```

Comment:
to be translated, you need to use 
```
"titleMap": [{ "name": "Swiss franc", "value": "CHF"},
{ "name": "Euro", "value": "EUR"},
{ "name": "American Dollar", "value": "USD"}]
```
with correct indentation

Reply:
Fixed